### ISSUE TITLE
Example only shows the model's hair

### ISSUE THREAD

**Issue by JeroenDerks** [bornfree/talking_avatar#5]
Hi, 

Thank you for a nice example on integrating the text to audio.

I can receive the audio and Viseme, but the model is not loading correctly.

Would you be so kind to show me how to display the entire model?

![Scherm­afbeelding 2023-05-04 om 10 06 11](https://user-images.githubusercontent.com/24194818/236146314-b99e469a-c989-471e-b909-0c0d8057021b.png)

**Comment by dmicheletto** [comment]
Hi @JeroenDerks 

I had your same problem. I solved it like this:
- delete the project from your pc
- check node, npm and yarn installation (my setup is node v20.2.0, npm 9.6.6 and yarn 1.22.19)
- download the project again
- run
$yarn install
$yarn start

**Comment by Sun-Deep** [comment]
Hi @dmicheletto,

I tried your suggested solution but the problem is still there. Any idea what may be the issue here? I am using Next.JS v13

**Comment by sumitsarraf** [comment]
> Hi @JeroenDerks
> 
> I had your same problem. I solved it like this:
> 
> * delete the project from your pc
> * check node, npm and yarn installation (my setup is node v20.2.0, npm 9.6.6 and yarn 1.22.19)
> * download the project again
> * run
>   $yarn install
>   $yarn start

![Screenshot 2023-07-26 103646](https://github.com/bornfree/talking_avatar/assets/34721357/e1b10a48-ed48-46fe-9431-e6d793abb324)

@dmicheletto @bornfree Same issue for me also. tried updating everything but nothing worked. any suggestion?

**Comment by mquenumsiax** [comment]
Same here !
![image](https://github.com/bornfree/talking_avatar/assets/129774627/c6ab71f8-693b-45ee-ac1d-f533cb1af43e)
Any other idea ?

**Comment by JeroenDerks** [comment]
I didnt manage to fix this example either. Decided to move on.

**Comment by sumitsarraf** [comment]
This is a really great sample code to start with. But the question is, can
we replace the 3D model with something else? If yes, please elaborate on
what type or kind of model I can use and where I can get it. Any
suggestions or guidance are highly appreciated.

and I'm trying to fix the same code if it starts working, I'll share the
repo with you soon


Thanks,
Sumit

**Comment by PatrickkZhao** [comment]
I fixed it by updating the latest library, also removing the Bg component.
` yarn add @react-three/drei @react-three/fiber @types/three three`

**Comment by Shivamkumar285** [comment]
> This is a really great sample code to start with. But the question is, can we replace the 3D model with something else? If yes, please elaborate on what type or kind of model I can use and where I can get it. Any suggestions or guidance are highly appreciated. and I'm trying to fix the same code if it starts working, I'll share the repo with you soon Thanks, Sumit
> […](#)

Hey did you get anything on this?

**Comment by Affan-Khan-13** [comment]
Hey anyone solved this ? @JeroenDerks @sumitsarraf @dmicheletto @Sun-Deep @mquenumsiax @PatrickkZhao 
I tried updating to latest versions, which just crashes the app instead

**Comment by daz-williams** [comment]
I had the same problem. Here's how I fixed it:

1. Clone repo
2. `yarn install`
3. `yarn add three`
4. Edit `App.js` change `<planeBufferGeometry />` to `<planeGeometry />`
5. `yarn start`

Three is the only package I updated.

Prior:  "`three": "^0.142.0",`

Updated to:  `"three": "^0.158.0",`


@JeroenDerks @sumitsarraf @dmicheletto @Sun-Deep @mquenumsiax @PatrickkZhao @Affan-Khan-13 @bornfree

**Comment by Affan-Khan-13** [comment]
> I had the same problem. Here's how I fixed it:
> 
> 1. Clone repo
> 2. `yarn install`
> 3. `yarn add three`
> 4. Edit `App.js` change `<planeBufferGeometry />` to `<planeGeometry />`
> 5. `yarn start`
> 
> Three is the only package I updated.
> 
> Prior: "`three": "^0.142.0",`
> 
> Updated to: `"three": "^0.158.0",`
> 
> @JeroenDerks @sumitsarraf @dmicheletto @Sun-Deep @mquenumsiax @PatrickkZhao @Affan-Khan-13 @bornfree

Yeah the problem is with the use of package manager, npm dosent work, you have got to use yarn

**Comment by NavodPeiris** [comment]
@JeroenDerks @sumitsarraf @dmicheletto @Sun-Deep @mquenumsiax @PatrickkZhao @Affan-Khan-13 @bornfree
you guys maybe interested in an agentic version of this: https://github.com/NavodPeiris/agentic-avatars

this integrates with AI providers and great for conversations in web apps. It directly runs in browser!
